src/app/components/core/configuration-manager/configuration-group/configuration-group.component.css

```diff
@@ -4,18 +4,10 @@
   }
 }
 
-.config-div {
-  margin-top: 1vw;
-}
-
-.tooltip.is-tooltip-multiline::before {
-  min-width: 16rem !important;
-}
-
```

Comment:
Why these have been removed?

Reply:
these classes are not required, as these are not being used anymore